source/manual/alerts/publisher-unprocessed-fact-check-emails.html.md

```diff
@@ -1,63 +1,83 @@
 
-If this alert is triggered, we want to [log into the fact-check inbox](login)
-and investigate why the emails were not processed and deleted.
+[login]: https://support.google.com/accounts/answer/1721977?co=GENIE.Platform%3DDesktop&hl=en
+
+The current email addresses used for the fact checking process are:
+
+- factcheck@alphagov.co.uk
+- govuk-fact-check-staging@digital.cabinet-office.gov
+- govuk-fact-check-integration@digital.cabinet-office.gov
+
+#### Retrieving credentials
 
-The password to log into the govuk-fact-check@digital.cabinet-office.gov.uk
-inbox can be found by running:
+The passwords for the @digital.cabinet-office.gov.uk addresses are found in
+[govuk-secrets]:
 
-```bash
+[govuk-secrets]: https://github.com/alphagov/govuk-secrets
+
+```sh
 PASSWORD_STORE_DIR=~/govuk/govuk-secrets/pass/2ndline pass google-accounts/govuk-fact-check@digital.cabinet-office.gov.uk
 ```
 
-The password to log into the factcheck@alphagov.co.uk inbox can be found by
-logging into the Publisher console:
+The password for the factcheck@alphagov.co.uk address can be found by logging
+into the Publisher console:
 
-```bash
-gds govuk connect app-console -e production publisher
+```sh
+$ gds govuk connect app-console -e production publisher
 ```
 
-and then running:
-
-```bash
-Publisher::Application.mail_fetcher_config
+```ruby
+> Publisher::Application.mail_fetcher_config
 ```
 
-The same can be done for
-staging (govuk-fact-check-staging@digital.cabinet-office.gov.uk)
-and integration (govuk-fact-check-integration@digital.cabinet-office.gov.uk).
+### Investigate the unprocessed emails
 
-[fact-check]: https://github.com/alphagov/publisher/blob/d0ab32c10c5d22ffb9b6edccb84f5345bd766cf4/doc/fact-checking.md
-[script]: https://github.com/alphagov/publisher/blob/d0ab32c10c5d22ffb9b6edccb84f5345bd766cf4/script/mail_fetcher
-[schedule]: https://github.com/alphagov/publisher/blob/d0ab32c10c5d22ffb9b6edccb84f5345bd766cf4/config/schedule.rb#L7-L9
-[process]: https://github.com/alphagov/publisher/blob/d0ab32c10c5d22ffb9b6edccb84f5345bd766cf4/lib/fact_check_email_handler.rb#L41-L53
-[warning]: https://github.com/alphagov/govuk-puppet/blob/68507dba280dd6e2fd4e5663f915c0fd7be06bef/modules/govuk/manifests/apps/publisher/unprocessed_emails_count_check.pp#L19
-[critical]: https://github.com/alphagov/govuk-puppet/blob/68507dba280dd6e2fd4e5663f915c0fd7be06bef/modules/govuk/manifests/apps/publisher/unprocessed_emails_count_check.pp#L20
-[login]: https://support.google.com/accounts/answer/1721977?co=GENIE.Platform%3DDesktop&hl=en
-[alert]: https://github.com/alphagov/govuk-puppet/blob/68507dba280dd6e2fd4e5663f915c0fd7be06bef/modules/govuk/manifests/apps/publisher/unprocessed_emails_count_check.pp
+There are a number of different ways of proceeding here based on why the email
+wasn't able to be processed:
+
+#### Spam/out of office
+
+If the email is clearly spam and not relevant, **it can be deleted**.
```

Comment:
```suggestion
If the email is clearly spam or not relevant, **it can be deleted**.
```
I wouldn't consider out of office emails as spam, but they are indeed not relevant.

Reply:
Ah yep, that's wrong. I've reworded it to make it clearer!